core/src/Kokkos_View.hpp

```diff
@@ -494,17 +494,11 @@ constexpr bool is_assignable(const Kokkos::View<ViewTDst...>& dst,
 
 namespace Kokkos {
 
-namespace {
+inline constexpr Kokkos::ALL_t ALL{};
 
-constexpr Kokkos::Impl::ALL_t ALL = Kokkos::Impl::ALL_t();
+inline constexpr Kokkos::Impl::WithoutInitializing_t WithoutInitializing{};
 
-constexpr Kokkos::Impl::WithoutInitializing_t WithoutInitializing =
```

Comment:
How was this ever usable by the public if it was in an anonymous namespace?

Reply:
just one per TU

Reply:
anonymous namespace means no collision at link time